README.md

```diff
@@ -52,6 +52,33 @@ brew install libmagic
 brew link libmagic  (if the link is already created is going to fail, don't worry about that)
 ```
 
+Additionally, problems have been reported on Windows when installing Submit4DN
+inside a virtual environment, due to `aws` trying to use the global python instead
+of the python inside the virtual environment.
+
+The workaround, then, because it’s actually OK if `aws` doesn’t use the python
+inside the virtual environment, is to just install `awscli` in the global
+environment before entering the virtual environment. Or if you discover the
+problem after you’re in, then go outside, install `awscli`, and re-enter the
+virtual environment.
+
+```
+deactivate
+pip install awscli
+venv\scripts\activate
+aws --version  # this is to test that awscli is now installed correctly
+```
+
+Alternatively, it works also to modify `aws.cmd`. Run:
+
+```
+notepad venv\scripts\aws.cmd
+```
+
+and change the for loop in the first couple of lines where it loops over
+`cmd bat exe` to have it loop over `exe bat cmd` instead, so that it finds
+`python.exe` before it finds `python.cmd`.
```

Comment:
Let's omit this solution for now and hold it in reserve. The problem with this is that the AWS team might feel like it's hard to maintain tools of theirs if we're editing them. I do plan to submit a bug report to them in which I make the suggestion that _they_ might want to use this approach themselves.

Reply:
OK I see, makes sense. I removed this.